docs/src/sources-sprites.md

```diff
@@ -40,6 +41,19 @@ the PNG, there is a high DPI version available at `/sprite/<sprite_id>@2x.json`.
 }
 ```
 
+##### Coloring at runtime via Signed Distance Fields (SDFs)
```

Comment:
> I need to make this more clear in the docs, why one would choose the one or the other image format

@hiddewie (hope you don't mind the ping too much)
Could you have a look at the docs if there is something that I missed?

Reply:
This looks great, thanks!